index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300&family=Poppins:wght@400;500;600;700;800&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css" integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.3.0/css/font-awesome.css" integrity="sha512-XJ3ntWHl40opEiE+6dGhfK9NAKOCELrpjiBRQKtu6uJf9Pli8XY+Hikp7rlFzY4ElLSFtzjx9GGgHql7PLSeog==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="style.css">
+  <title>Document</title>
+</head>
+<body>
+  <main class="header-bg">
+    <nav id="appBar">
+      <div id="toolsBar">
+        <div class="logoPlaceHolder">
+          <h1 class="logo"> My Logo</h1>
+        </div>
+        <div id="Hamburger">
+          <div class="union">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+          </div>
+        
+        </div>
+
+      </div>
+    </nav>
+    
+    <div class="body-shae">
+      <div class="headline">
+        <h1 
+          class="headerText">
+          I'm Adeyemi Glad to meet you!
+        </h1>
+
+        <p 
+          class="headerbodytext">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+       
+        <div class="social">
+          <h2 class="connect">LET'S CONNECT</h2>
+        </div>
+        <i class="icons">
+          <i class="fa-brands fa-twitter"></i>
+          <i class="fa-brands fa-github"></i>
+          <i class="fa-brands fa-linkedin-in"></i>
+          <i class=" fa-brands fa-medium"></i>
+          <i class="fa-brands fa-angellist"></i>
+          
+        </i>
+          
+      </div>
+      
+    </div>
```

Comment:
- [x] Kindly make sure to wrap the headline container into the `section` tag instead of using `headline`. It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.